test/UnitTest/v1/test_casesNGSIv1.py

```diff
@@ -16,704 +16,974 @@
         resp=json.loads(resInJson)
-        #print(resp)
+        print(resp)
         resp=resp['subscribeResponse']
         sid=resp['subscriptionId']
-        #print(sid)
+        print(sid)
 
-	#update the created entity
+        #update the created entity
         url=brokerIp+"/ngsi10/updateContext"
         r=requests.post(url,data=json.dumps(data_ngsi10.subdata20),headers=headers)
-        #print(r.status_code)
         resp_content1=r.content
         resInJson=resp_content1.decode('utf8').replace("'",'"')
         resp1=json.loads(resInJson)
-        #print(resp1)
+        print(resp1)
 
-	# validate via accumulator
-        url="http://0.0.0.0:8888/validateNotification"
+        #validate via accumulator
+        url="http://180.179.214.211:8888/validateNotification"
         r=requests.post(url,json={"subscriptionId" : sid})
         print(r.content)
         assert r.status_code == 200
-        
+
 #testCase 12
 '''
-  Testing subscription for its if and else part : 1) for Destination Header
+  To test subscription for its if and else part : 1) for Destination Header
 '''
 def test_getSubscription12():
-	#create an entity
+        #create an entity
         url=brokerIp+"/ngsi10/updateContext"
         headers={'Content-Type' : 'application/json'}
-	r=requests.post(url,data=json.dumps(data_ngsi10.subdata21),headers=headers)
-        #print(r.status_code)
+        r=requests.post(url,data=json.dumps(data_ngsi10.subdata21),headers=headers)
         resp_content=r.content
         resInJson=resp_content.decode('utf8').replace("'",'"')
         resp=json.loads(resInJson)
-	#print(resp)
+        print(resp)
 
-	#subscribing
-	url=brokerIp+"/ngsi10/subscribeContext"
+        #subscribing
+        url=brokerIp+"/ngsi10/subscribeContext"
         headers= {'Content-Type': 'application/json','Destination' : 'orion-broker'}
         r=requests.post(url,data=json.dumps(data_ngsi10.subdata22),headers=headers)
-        #print(r.status_code)
         resp_content=r.content
         resInJson= resp_content.decode('utf8').replace("'", '"')
         resp=json.loads(resInJson)
-        #print(resp)
+        print(resp)
         resp=resp['subscribeResponse']
         sid=resp['subscriptionId']
-        #print(sid)
+        print(sid)
 
-	#update the created entity
+        #update the created entity
         url=brokerIp+"/ngsi10/updateContext"
         r=requests.post(url,data=json.dumps(data_ngsi10.subdata23),headers=headers)
-        #print(r.status_code)
         resp_content1=r.content
         resInJson=resp_content1.decode('utf8').replace("'",'"')
         resp1=json.loads(resInJson)
-        #print(resp1)
+        print(resp1)
 
-	#validate via accumulator
-        url="http://0.0.0.0:8888/validateNotification"
+        #validate via accumulator
+        url="http://180.179.214.211:8888/validateNotification"
         r=requests.post(url,json={"subscriptionId" : sid})
         print(r.content)
         assert r.status_code == 200
-        
+
```

Comment:
For test case #12 and other similar test cases:
Subscriber should be orion-broker here, if it is mentioned in the header. Entity should be available on orion after successful notification of subscribed data. Also confirm the significance of 'User-Agent' : 'lightweight-iot-broker' in TC#13, whether this notification would be received by other thinBroker within FogFlow (i.e. internal communication)? Also correct the TC#15,18 accordingly if required, because if ThinBroker is lightweight-iot-broker, and orion is the destination, both cannot be simultaneously true.